src/main/resources/db/migration/V020__Add_ocrDataJson_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+ADD COLUMN ocrDataJson JSON null
```

Comment:
might be worth just naming the column as ocrData rather than having type in the column name

Reply:
ocrData is an existing column. @lgonczar is making non-breaking migration approach by just adding a new column